File: CS/WindowsFormsApplication1/XtraReport1.cs
using System;
using System.Drawing;
using System.Collections;
using System.ComponentModel;
using DevExpress.XtraReports.UI;

namespace WindowsFormsApplication1
{
    public partial class XtraReport1 : DevExpress.XtraReports.UI.XtraReport
    {
        public XtraReport1()
        {
            InitializeComponent();
            
        }

        private void xrPivotGrid1_BeforePrint(object sender, System.Drawing.Printing.PrintEventArgs e)
        {
            //nWindDataSet1.WriteXml("test.xml", System.Data.XmlWriteMode.WriteSchema);
            
           
        }

        private void XtraReport1_AfterPrint(object sender, EventArgs e)
        {
          
        }

        private void xrTableRow3_BeforePrint(object sender, System.Drawing.Printing.PrintEventArgs e)
        {

        }

        private void xrLabel1_SummaryRowChanged(object sender, EventArgs e) {

        }

        private void xrLabel1_SummaryCalculated(object sender, TextFormatEventArgs e) {

        }

        private void xrLabel1_BeforePrint(object sender, System.Drawing.Printing.PrintEventArgs e) {

        }

        private void xrLabel1_SummaryReset(object sender, EventArgs e) {
        }

        private void xrPivotGrid1_CustomCellValue(object sender, DevExpress.XtraPivotGrid.PivotCellValueEventArgs e) {
        
        }

    }
}

File: VB/WindowsFormsApplication1/XtraReport1.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Drawing
Imports System.Collections
Imports System.ComponentModel
Imports DevExpress.XtraReports.UI

Namespace WindowsFormsApplication1
	Partial Public Class XtraReport1
		Inherits DevExpress.XtraReports.UI.XtraReport
		Public Sub New()
			InitializeComponent()

		End Sub

		Private Sub xrPivotGrid1_BeforePrint(ByVal sender As Object, ByVal e As System.Drawing.Printing.PrintEventArgs)
			'nWindDataSet1.WriteXml("test.xml", System.Data.XmlWriteMode.WriteSchema);


		End Sub

		Private Sub XtraReport1_AfterPrint(ByVal sender As Object, ByVal e As EventArgs) Handles MyBase.AfterPrint

		End Sub

		Private Sub xrTableRow3_BeforePrint(ByVal sender As Object, ByVal e As System.Drawing.Printing.PrintEventArgs)

		End Sub

		Private Sub xrLabel1_SummaryRowChanged(ByVal sender As Object, ByVal e As EventArgs)

		End Sub

		Private Sub xrLabel1_SummaryCalculated(ByVal sender As Object, ByVal e As TextFormatEventArgs)

		End Sub

		Private Sub xrLabel1_BeforePrint(ByVal sender As Object, ByVal e As System.Drawing.Printing.PrintEventArgs)

		End Sub

		Private Sub xrLabel1_SummaryReset(ByVal sender As Object, ByVal e As EventArgs)
		End Sub

		Private Sub xrPivotGrid1_CustomCellValue(ByVal sender As Object, ByVal e As DevExpress.XtraPivotGrid.PivotCellValueEventArgs)

		End Sub

	End Class
End Namespace
